index.bs

```diff
@@ -2669,10 +2674,18 @@ The <dfn export for=commands>browser.createUserContext</dfn> command creates a
 
 <div algorithm="remote end steps for browser.createUserContext">
 
-The [=remote end steps=] are:
+The [=remote end steps=] with |command parameters| are:
 
 1. Let |user context| be a new [=user context=].
 
+1. If |command parameters| [=map/contains=] "<code>proxy</code>":
+
+   1. Let |proxy| be [=deserialize as a proxy=] with |command parameters|["<code>proxy</code>"].
+
+   1. Take implementation-defined steps to set the |user context|'s proxy using the
```

Comment:
Let's rephrase to that we take implementation-defined steps to ensure that requests originating from the user context use the proxy defined by the proxy configuration? `set the |user context|'s proxy using` does not sound precise.

Reply:
done